packages/layout/src/ChartPanel.ts

```diff
@@ -554,6 +587,7 @@ ChartPanel.prototype.publishProxy("progress_blurBar", "_progressBar", "blurBar")
 ChartPanel.prototype.publishProxy("progress_blurSize", "_progressBar", "blurSize");
 ChartPanel.prototype.publishProxy("progress_blurColor", "_progressBar", "blurColor");
 ChartPanel.prototype.publishProxy("progress_blurOpacity", "_progressBar", "blurOpacity");
+ChartPanel.prototype.publish("legendPosition", "right", "set", "Position of legend (must be set before initial render)", ["right", "bottom"]);
```

Comment:
Move PP so its with the other legend items
Rename so its consistent with the other legend items